modules/relabeling-selinux-workaround.adoc

```diff
@@ -0,0 +1,57 @@
+
+When attempting to migrate a namespace with {mtc-full} ({mtc-short}) and a substantial volume associated with it, the `rsync-server` may become frozen without any further information to troubleshoot the issue.
+
+[id="diagnosis-selinux-workaround_{context}"]
+== Diagnosing the need for the Skip SELinux relabel workaround
+
+Search for an error of `Unable to attach or mount volumes for pod...timed out waiting for the condition` in the kubelet logs from the node where the `rsync-server` for the Direct Volume Migration (DVM) runs.
+
+.Example kubelet log
+[source,yaml]
+----
+kubenswrapper[3879]: W0326 16:30:36.749224    3879 volume_linux.go:49] Setting volume ownership for /var/lib/kubelet/pods/8905d88e-6531-4d65-9c2a-eff11dc7eb29/volumes/kubernetes.io~csi/pvc-287d1988-3fd9-4517-a0c7-22539acd31e6/mount and fsGroup set. If the volume has a lot of files then setting volume ownership could be slow, see https://github.com/kubernetes/kubernetes/issues/69699
+
+kubenswrapper[3879]: E0326 16:32:02.706363    3879 kubelet.go:1841] "Unable to attach or mount volumes for pod; skipping pod" err="unmounted volumes=[8db9d5b032dab17d4ea9495af12e085a], unattached volumes=[crane2-rsync-server-secret 8db9d5b032dab17d4ea9495af12e085a kube-api-access-dlbd2 crane2-stunnel-server-config crane2-stunnel-server-secret crane2-rsync-server-config]: timed out waiting for the condition" pod="caboodle-preprod/rsync-server"
+
+kubenswrapper[3879]: E0326 16:32:02.706496    3879 pod_workers.go:965] "Error syncing pod, skipping" err="unmounted volumes=[8db9d5b032dab17d4ea9495af12e085a], unattached volumes=[crane2-rsync-server-secret 8db9d5b032dab17d4ea9495af12e085a kube-api-access-dlbd2 crane2-stunnel-server-config crane2-stunnel-server-secret crane2-rsync-server-config]: timed out waiting for the condition" pod="caboodle-preprod/rsync-server" podUID=8905d88e-6531-4d65-9c2a-eff11dc7eb29
+----
+
+[id="resolving-selinux-workaround_{context}"]
+== Resolving using the Skip SELinux relabel workaround
+
+To resolve this issue, set the `migration_rsync_super_privileged` parameter to `true` in both the source and destination `MigClusters` using the `MigrationController` custom resource (CR).
+
+.Example MigrationController CR
+
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigrationController
+metadata:
+  name: migration-controller
+  namespace: openshift-migration
+spec:
+  migration_rsync_super_privileged: true # <1>
+  azure_resource_group: ""
+  cluster_name: host
+  mig_namespace_limit: "10"
+  mig_pod_limit: "100"
+  mig_pv_limit: "100"
+  migration_controller: true
+  migration_log_reader: true
+  migration_ui: true
+  migration_velero: true
+  olm_managed: true
+  restic_timeout: 1h
+  version: 1.8.3
+----
+
+<1> `migration_rsync_super_privileged` indicates whether to run Rsync Pods as _super privileged_ containers (`spc_t selinux context`), or not.
```

Comment:
```suggestion
<1> The value of the `migration_rsync_super_privileged` parameter indicates whether or not to run Rsync Pods as _super privileged_ containers (`spc_t selinux context`).
```
I think this is what you mean, assuming this is a `true` or `false` type of entry. You could actually give the values in the footnote if it's only two possibilities.